src/Neo/Hardfork.cs

```diff
@@ -13,6 +13,7 @@ namespace Neo
     public enum Hardfork : byte
     {
         HF_Aspidochelone,
-        HF_Basilisk
+        HF_Basilisk,
+        HF_Manticore
```

Comment:
Can't be.

Reply:
Centaurus!

Reply:
https://github.com/neo-project/neo/pull/2749#discussion_r878971641

Sorry.

Reply:
Chat-Gpt said `Manticore` xD

Reply:
```suggestion
        HF_Cockatrice
```